# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# Ani Peter <apeter@redhat.com>, 2012-2013
# Jiří Moskovčák <jmoskovc@redhat.com>, 2011
msgid ""
msgstr ""
"Project-Id-Version: gnome-abrt\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2013-05-02 09:30+0200\n"
"PO-Revision-Date: 2013-05-02 07:30+0000\n"
"Last-Translator: Jakub Filak <jfilak@redhat.com>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: ml\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: ../src/gnome-abrt.desktop.in.h:1 ../src/gnome_abrt/views.py:155
msgid "Automatic Bug Reporting Tool"
msgstr "ഓട്ടോമാറ്റിക് ബഗ് രേഖപ്പെടുത്തല്‍ പ്രയോഗം"

#: ../src/gnome-abrt.desktop.in.h:2
msgid "View and report application crashes"
msgstr "പ്രയോഗത്തിന്റെ തകരാറുകള്‍ രേഖപ്പെടുത്തുക"

#: ../src/gnome_abrt/dbus_problems.py:55
msgid "Can't add receiver of signal '{0}'on DBus system path '{1}': {2}"
msgstr "'{0}' സിഗ്നലിന്റെ റിസീവര്‍ '{1}' എന്ന ഡീബസ് സിസ്റ്റം പാഫില്‍ ചേര്‍ക്കുവാന്‍ സാധ്യമല്ല: {2}"

#: ../src/gnome_abrt/dbus_problems.py:78
msgid "Can't connect to DBus system bus '{0}' path '{1}': {2}"
msgstr "'{0}' ഡിബസ് സിസ്റ്റം ബസ് '{1}' പാഥിലേക്കു് കണക്ട് ചെയ്യുവാന്‍ സാധ്യമല്ല: {2}"

#: ../src/gnome_abrt/dbus_problems.py:85
msgid "Can't get interface '{0}' on path '{1}' in bus '{2}': {3}"
msgstr "'{0}' ഇന്റര്‍ഫെയിസ്, '{1}' എന്ന പാഥിലുള്ള '{2}' ബസില്‍ ലഭ്യമായില്ല: {3}"

#: ../src/gnome_abrt/dbus_problems.py:139
msgid "Can't chown problem '{0}' over DBus service: {1!s}"
msgstr "ഡിബസ് സര്‍വീസില്‍ '{0}' പ്രശ്നം ചൌണ്‍ ചെയ്യുവാന്‍ സാധ്യമല്ല.: {1!s}"

#: ../src/gnome_abrt/dbus_problems.py:160
msgid "Can't get problem data from DBus service: {0!s}"
msgstr "ഡിബസ് സര്‍വീസില്‍ നിന്നുള്ള തകരാറുള്ള ഡേറ്റാ ലഭ്യമാക്കുവാന്‍ സാധ്യമല്ല: {0!s}"

#: ../src/gnome_abrt/dbus_problems.py:181
msgid "Can't get list of problems from DBus service: {0!s}"
msgstr "ഡീബസ് സര്‍വീസില്‍ നിന്നുള്ള പ്രശ്നങ്ങളുടെ പട്ടിക ലഭ്യമാക്കുവാന്‍ സാധ്യമല്ല: {0!s}"

#: ../src/gnome_abrt/dbus_problems.py:199
msgid "Can't delete problem over DBus service: {0!s}"
msgstr "ഡിബസ് സര്‍വീസിലുള്ള പ്രശ്നം വെട്ടി നീക്കുവാന്‍ സാധ്യമല്ല: {0!s}"

#: ../src/gnome_abrt/dialogs.py:50
msgid "The description of problem is too short"
msgstr ""

#: ../src/gnome_abrt/dialogs.py:51
msgid ""
"In order to get more useful reports we do not accept reports with the "
"description shorter than 10 letters."
msgstr ""

#: ../src/gnome_abrt/dialogs.py:61
msgid "The summary of problem is too short"
msgstr ""

#: ../src/gnome_abrt/dialogs.py:62
msgid ""
"In order to get more useful reports we do not accept reports with the "
"summary shorter than 10 letters."
msgstr ""

#: ../src/gnome_abrt/dialogs.py:88
msgid "Problem description"
msgstr ""

#: ../src/gnome_abrt/dialogs.py:96
msgid "Summary:"
msgstr ""

#: ../src/gnome_abrt/dialogs.py:108
msgid "_Send"
msgstr ""

#. Sets text but text is selected :)
#: ../src/gnome_abrt/dialogs.py:122
msgid "A problem with ABRT"
msgstr ""

#: ../src/gnome_abrt/directory_problems.py:206
msgid ""
"You have probably reached inotify's limit on the number of watches in '{0}'."
" The limit can be increased by proper configuration of inotify. For more "
"details see man inotify(7). This event causes that you will not be notified "
"about changes in problem data happening outside of this application. This "
"event do not affect any other functionality."
msgstr ""

#: ../src/gnome_abrt/directory_problems.py:252
msgid "Can't open directory: '{0}'"
msgstr "ഡയറക്ടറി തുറക്കുവാന്‍ സാധ്യമല്ല: '{0}'"

#: ../src/gnome_abrt/oops.glade.h:1
msgid "Delete"
msgstr "വെട്ടി നീക്കുക"

#: ../src/gnome_abrt/oops.glade.h:2
msgid "Delete selected problems"
msgstr "തെരഞ്ഞെടുത്ത പ്രശ്നങ്ങള്‍ വെട്ടി നീക്കുക"

#: ../src/gnome_abrt/oops.glade.h:3
msgid "_Report"
msgstr "_രേഖപ്പെടുത്തുക"

#: ../src/gnome_abrt/oops.glade.h:4
msgid "Submits selected problem"
msgstr "തെരഞ്ഞെടുത്ത പ്രശ്നം സമര്‍പ്പിയ്ക്കുന്നു"

#: ../src/gnome_abrt/oops.glade.h:5
msgid "Analy_ze"
msgstr ""

#: ../src/gnome_abrt/oops.glade.h:6
msgid "Open selected problem for analysis"
msgstr ""

#: ../src/gnome_abrt/oops.glade.h:7
msgid "_All problems"
msgstr "_എല്ലാ പ്രശ്നങ്ങളും"

#: ../src/gnome_abrt/oops.glade.h:8
msgid "Show all known problems from all system users"
msgstr "എല്ലാ ഉപയോക്താക്കളില്‍ നിന്നുള്ള പ്രശ്നങ്ങള്‍ കാണിയ്ക്കുക"

#: ../src/gnome_abrt/oops.glade.h:9
msgid "_Open problem's data directory"
msgstr ""

#: ../src/gnome_abrt/oops.glade.h:10
msgid ""
"Opens the selected problem's data directory in the default file system "
"browser"
msgstr ""

#: ../src/gnome_abrt/oops.glade.h:11
msgid "_Copy problem's ID to Clipboard"
msgstr ""

#: ../src/gnome_abrt/oops.glade.h:12
msgid "Stores the selected problem's ID in Clibpoard"
msgstr ""

#: ../src/gnome_abrt/oops.glade.h:13
msgid "Name"
msgstr "പേരു്"

#: ../src/gnome_abrt/oops.glade.h:14
msgid "Version"
msgstr "പതിപ്പു്"

#: ../src/gnome_abrt/oops.glade.h:15
msgid "Detected"
msgstr ""

#: ../src/gnome_abrt/oops.glade.h:16
msgid "Reported"
msgstr "സമര്‍പ്പിച്ചു"

#: ../src/gnome_abrt/oops.glade.h:17
msgid "No problems detected!"
msgstr ""

#: ../src/gnome_abrt/oops.glade.h:18
msgid "App"
msgstr "പ്രയോഗം"

#: ../src/gnome_abrt/oops.glade.h:19
msgid "Date"
msgstr "തീയതി"

#: ../src/gnome_abrt/problems.py:174
msgid "Can't delete problem '{0}': '{1}'"
msgstr "'{0}' പ്രശ്നം വെട്ടി നീക്കുവാന്‍ സാധ്യമല്ല: '{1}'"

#: ../src/gnome_abrt/problems.py:345
msgid "Not found in cache but deleted: {0}"
msgstr "കാഷില്‍ ലഭ്യമല്ല, പക്ഷേ വെട്ടി നീക്കിയിരിയ്ക്കുന്നു: {0}"

#: ../src/gnome_abrt/problems.py:364
msgid "Source failed on processing of '{0}': {1}"
msgstr "'{0}'-ന്റെ പ്രക്രിയയില്‍ സോഴ്സ് പരാജയപ്പെട്ടു: {1}"

#: ../src/gnome_abrt/tools.py:33
msgid "Future"
msgstr "ഭാവി"

#: ../src/gnome_abrt/tools.py:40
msgid "Yesterday"
msgstr "ഇന്നലെ"

#: ../src/gnome_abrt/tools.py:49
msgid "week"
msgstr "ആഴ്ച"

#: ../src/gnome_abrt/tools.py:52
msgid "month"
msgstr "മാസം"

#: ../src/gnome_abrt/tools.py:55
msgid "year"
msgstr "വര്‍ഷം"

#: ../src/gnome_abrt/tools.py:58
msgid "Last {0!s}"
msgstr "അവസാന {0!s}"

#: ../src/gnome_abrt/tools.py:60
msgid "{0:d} {1!s}s ago"
msgstr "{0:d} {1!s}s മുമ്പു്"

#: ../src/gnome_abrt/views.py:83 ../src/gnome_abrt/views.py:368
#: ../src/gnome_abrt/views.py:370
msgid "N/A"
msgstr "ലഭ്യമല്ല"

#: ../src/gnome_abrt/views.py:370
msgid " crashed"
msgstr "തകരാറും സംഭവിച്ചു"

#: ../src/gnome_abrt/views.py:383
msgid "yes"
msgstr "ഉവ്വു്"

#: ../src/gnome_abrt/views.py:386
msgid "no"
msgstr "ഇല്ല"

#: ../src/gnome_abrt/views.py:394
msgid ""
"This problem hasn't been reported to <i>Bugzilla</i> yet. Our developers may need more information to fix the problem.\n"
"Please consider <b>reporting it</b> - you may help them. Thank you."
msgstr ""
